src/index.tsx

```diff
@@ -161,6 +164,11 @@ const ModalizeBase = (
   const modalOverlay = useRef<PanGestureHandler>(null);
   const modalOverlayTap = useRef<TapGestureHandler>(null);
 
+  if (scrollViewRef) {
```

Comment:
Could we instead use something like this https://github.com/seznam/compose-react-refs here https://github.com/jeremybarbet/react-native-modalize/blob/master/src/index.tsx#L600

and do something like:

```tsx
ref: composeRef(scrollViewProps?.ref, contentView),
```

It could be nice to have the compose util locally in the util folder instead of adding an extra dependency 👍